src/main/java/com/taskadapter/redmineapi/internal/RedmineDateUtils.java

```diff
@@ -1,9 +1,14 @@
 package com.taskadapter.redmineapi.internal;
 
+import java.text.ParseException;
+import java.util.Date;
+import javax.xml.bind.DatatypeConverter;
```

Comment:
this won't work on Android and who knows where else. this library has nothing to do with XML parsing and we should not use classes that "accidentally" do what we need.